raphtory-graphql/src/model/graph/edge.rs

```diff
@@ -56,85 +58,62 @@ impl GqlEdge {
     ////////////////////////
 
     async fn default_layer(&self) -> GqlEdge {
-        let self_clone = self.clone();
-        spawn_blocking(move || self_clone.ee.default_layer().into())
-            .await
-            .unwrap()
+        self.ee.default_layer().into()
```

Comment:
Does this always return something now or should it be an option?

Reply:
it currently always returns a view that might be empty